### ISSUE TITLE
Selecting amiibo does not enable them by default

### ISSUE THREAD

**Issue by bell07** [FuryBaguette/AmiiSwap#14]
I updated the Emuiibo and AmiiSwap on my switch last days and all is working fine. Except one behaviour.
After Switch-Reboot the emuiibo is disabled by default. In old version I just entered AmiiSwap and selected amiibo, the Emuiibo was enabled automatically. Also 
1. use AmiiSwap
2. call the game
3. use the Emuiibo.

Now the AmiiSwap  does not enable the Emuiibom that results in 
1. use AmiiSwap
2. call the game
3. find out the Emuiibo is disabled
4. leave the game
5. use emuiibo toggle combo
6. call the game again
7. use the Emuiibo

In short: please enable the Emuiibo if an Amiibo is selected in AmiiSwap.

An other nice feature would be to see which Amiibo is currently selected and the status if Amiibo is enabled in AmiiSwap.

**Comment by AD2076** [comment]
In the next version you'll be able to see and change emuiibo status: enabled, enabled once, disabled.
